### PR TITLE
[WIP] Sprout - enhanced logging, new API method

### PR DESCRIPTION
**Pull request by mfalesni**

### CODE REVIEW

File: sprout/appliances/models.py [hunk 1]
```diff
@@ -368,6 +374,24 @@ class Power(object):
         help_text="Appliance has an IP address and web UI is online.")
     uuid = models.CharField(max_length=36, null=True, blank=True, help_text="UUID of the machine")
 
+    def serialized(self):
```
Comment (psav) [inline sprout/appliances/models.py]: should this be a property?

Comment (mfalesni) [inline sprout/appliances/models.py]: Ok.

**Comment by dajoRH** [comment]
Lint report for commit 4d45f7a394afb8a3bb55ac741c82036c944cda2c:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by psav** [comment]
Looks good :cake: :punch:

**Pull request merged**